arrays/index.js

```diff
@@ -0,0 +1,20 @@
+// Hobbies
+const hobbies = new Array("water skiing", "bouldering", "running",)
+
+console.log(hobbies.shift())
+
+const firstHobby = hobbies.shift()
+
+const arrLength = hobbies.length
+
+hobbies.push("Soccer")
+
+console.log(hobbies.pop())
```

Comment:
This will remove the last element of the array thus mutating the array. If we want to access the last element without mutating we could do hobbies[hobbies.length - 1]